video_frame_metadata_registry.src.html

```diff
@@ -1,7 +1,7 @@
 <pre class='metadata'>
 Title: WebCodecs VideoFrame Metadata Registry
 Repository: w3c/webcodecs
-Status: DRY
+Status: ED
```

Comment:
Why this change?

Reply:
Although it has "Draft" in the title, `DRY` is a status for documents published under /TR. All documents in the repo are "Editor's Drafts". There is no `DRY-ED` status in authoring tools for now to qualify an "Editor's Draft of a Registry", so using "ED" as fallback (I see the status of the Editor's Draft of the Codec registry is `NOTE-ED` for now, that's really treated the same way as `ED` internally and produces the same boilerplate).